libcpuid/libcpuid_types.h

```diff
@@ -58,8 +58,12 @@ typedef unsigned short		uint16_t;
 	typedef unsigned __int64	uint64_t;
 #else
 	/* all other sane compilers: */
-	typedef signed long long   int64_t;
-	typedef unsigned long long uint64_t;
+	#if __WORDSIZE == 64
+	typedef unsigned long int   uint64_t;
```

Comment:
The original code defines both `int64_t` and `uint64_t`, but this suggestion only defines `uint64_t`.

Reply:
this one does as well... but changes define vars if using visual studio or other compiler...